torch/distributions/transformed_distribution.py

```diff
@@ -0,0 +1,77 @@
+import torch
+from numbers import Number
+from torch.autograd import Variable
+from torch.distributions.bijectors import Bijector
+from torch.distributions.distribution import Distribution
+
+
+class TransformedDistribution(Distribution):
+    r"""
+    Extension of the Distribution class, which applies a sequence of Bijectors to a base distribution.
+    Let f be the composition of bijectors applied,
+    X ~ BaseDistribution
+    Y = f(X) ~ TransformedDistribution(BaseDistribution, f)
+    log p(Y) = log p(X) + log det (dX/dY)
+    """
+    def __init__(self, base_distribution, bijectors=[], *args, **kwargs):

```

Comment:
I think it would be cleaner to remove `*args, **kwargs` and maybe flatten out bijectors into `*bijectors`. WDYT?